### PR TITLE
Starting the UI for NFT minting

### PR DESCRIPTION
**Pull request by dvargasfuertes**
- Added tab/router navigation
- Added a modal component, that I intend to now migrate to TOPO based on Mel's latest landing page design
- Next step is get a smart contract deployed on a test net

### Demo

passports1

### CODE REVIEW

File: packages/app/pages/index.tsx [hunk 1]
```diff
@@ -1,13 +1,197 @@
...
+    <div
+      style={{ minHeight: 64, padding: 20, cursor: "pointer" }}
+      onClick={onClick}
+    >
+      {children}
+    </div>
+  );
+};
+
+const Portal: React.FC = ({ children }) => <>{children}</>; // Replace with react-portal in TOPO
```
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:24]: If you notice in the demo, the modal backdrop was only contained within the tab content. Installing react-portal will make it full screen

File: packages/app/pages/index.tsx [hunk 2]
```diff
@@ -1,13 +1,197 @@
...
+  onConfirm?: () => void | Promise<void>;
+}> = ({
+  title,
+  isOpen,
+  setIsOpen,
+  children,
+  confirmText = "Submit",
+  onConfirm,
+}) => {
+  // TODO migrate to TOPO
```
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:40]: This whole component will be moved to TOPO. Should base components like this be created there first?

File: packages/app/pages/index.tsx [hunk 3]
```diff
@@ -1,13 +1,197 @@
...
+const UsersTabContent = () => {
+  return (
+    <>
+      <h1>Users</h1>
+      <div>Coming Soon!</div>
+    </>
+  );
+};
+
+const SettingsTabContent = () => {
```
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:183]: These tabs I selected were somewhat arbitrary. I envision us getting more clarity on what they should actually be as the product evolves

File: packages/app/pages/index.tsx [hunk 4]
```diff
@@ -89,25 +273,27 @@ const Home: NextPage = () => {
         >
           <div style={{ minHeight: 64, padding: 20 }}>
             <Link href="/">
-              <a>Passports</a>
+              <a>NFT Passports Dashboard</a>
             </Link>
           </div>
+          <Tab to={"memberships"}>Memberships</Tab>
+          <Tab to={"users"}>Users</Tab>
+          <Tab to={"settings"}>Settings</Tab>
```
Comment (dvargasfuertes) [inline packages/app/pages/index.tsx:281]: we could abstract these tabs out to a constant array and loop once we have more clarity on what they should be and if design even wants to stick with a sidebar-tabbed dashboard anyway

**Review by dvargasfuertes — COMMENTED** [review]
My process will be giving PRs 24 hours without comment before merging into main

**Review by 0xZakk — APPROVED** [review]
Looks fucking great

**Pull request merged**